quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt

```diff
@@ -62,71 +53,53 @@ object VitaminQuantityPickers {
         subtractEnabled: Boolean = true,
         editTextEnabled: Boolean = true,
         isExpanded: Boolean = false,
+        helperText: String? = null,
         keyboardOptions: KeyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
         keyboardActions: KeyboardActions = KeyboardActions.Default,
         colors: QuantityColors = VitaminQuantitiesColors.normal(),
-        shape: CornerBasedShape = VitaminTheme.shapes.radius100,
-        textStyle: TextStyle = VitaminTheme.typography.text2,
+        sizes: QuantitySizes = VitaminQuantitiesSizes.primary(),
         ripple: RippleTheme = VitaminTheme.ripples.brand,
         onAddClicked: () -> Unit,
         onSubtractClicked: () -> Unit,
         onValueChange: (String) -> Unit
     ) {
         CompositionLocalProvider(LocalRippleTheme provides ripple) {
-            Row(
+            val contentDescription = stringResource(id = R.string.vtmn_quantity_picker_accessibility_message, value)
+
+            Column(
                 modifier = modifier
-                    .height(48.dp)
-                    .then(
-                        if (isExpanded) {
-                            modifier.fillMaxWidth()
-                        } else {
-                            modifier.width(156.dp)
-                        }
-                    )
+                    .semantics(
+                        mergeDescendants = true
```

Comment:
From memory, if you configure a merge descendants, you can't access to internal components.
It means that you can't access to add and remove button with talk back?
If yes, we need to change this.

Reply:
With mergeDescendants, you can access to internal components.

https://user-images.githubusercontent.com/46709142/227148835-ca227b03-96da-403b-8b6b-f35296f686b4.mp4